build.sbt

```diff
@@ -3,6 +3,7 @@ import sbtassembly.AssemblyPlugin.defaultUniversalScript
 ThisBuild / scalaVersion := "2.13.6"
 ThisBuild / organization := "com.github.cross-language-cpp"
 
+val binExt = if (System.getProperty("os.name").startsWith("Windows")) ".bat" else ""
```

Comment:
Actually, we don't need to remove that line, as in the build automation, the generator is built on Ubuntu, and it will not add `.bat` file extension in that case, so we still need to copy the `target/bin/djinni` to `target/bin/djinni.bat`, so later we can upload `target/bin/djinni.bat` as the release asset for Windows. If you agree, then I will not change anything further.

Reply:
Ahh, right. Sry for the confusion...